Source/WebDriver/WebDriverService.cpp

```diff
@@ -29,19 +29,36 @@
 #include "Capabilities.h"
 #include "CommandResult.h"
 #include "SessionHost.h"
+#include <wtf/LoggerHelper.h>
 #include <wtf/RunLoop.h>
 #include <wtf/SortedArrayMap.h>
 #include <wtf/text/MakeString.h>
 #include <wtf/text/StringToIntegerConversion.h>
 #include <wtf/text/WTFString.h>
 
+#if ENABLE(WEBDRIVER_BIDI)
+#include "HTTPServer.h"
+#include "WebSocketServer.h"
+#include <algorithm>
```

Comment:
I would prefer we put WTF headers above system headers.
https://webkit.org/code-style-guidelines/#include-system

Reply:
I had tried that before, but `check-webkit-style` complained about the alphabetical ordering. In fact, the following `webkitpy` test fails with the alphabetical ordering error, suggesting `<wtf..>` being treated as regular system includes:

```
        self.assert_language_rules_check('foo.h',
                                         '#include <wtf/Function.h>\n'
                                         '#include <b>\n'
                                         '#include <c>\n',
                                         '')
```